packages/yew-macro/tests/html_lints/fail.stderr

```diff
@@ -1,9 +1,3 @@
-warning: The tag 'tExTAreA' is not matching its normalized form 'textarea'. If you want to keep this form, change this to a dynamic tag `@{"tExTAreA"}`.
-  --> tests/html_lints/fail.rs:17:10
-   |
-17 |         <tExTAreA />
-   |          ^^^^^^^^
-
```

Comment:
For some reason, the update to proc-macro-error2 crate made it so that this lint is triggered. Can you please investigate and see what's going on?

To re-generate the files, you can run the test command with TRYBUILD=overwrite

Reply:
I haven't been able to reproduce the issue on my machine yet. I'll try to work through this.

Reply:
Thanks! I'm going to go ahead and bisect the proc-macro-errors2 project.